data_import/serializers.py

```diff
@@ -43,3 +44,46 @@ def to_representation(self, instance):
         ret["source"] = instance.source
 
         return ret
+
+
+class NewDataFileAccessLogSerializer(serializers.ModelSerializer):
+    """
+    Serialize the logs of file access events from Open Human's end.
+    """
+
+    user = serializers.IntegerField(source="user.id", allow_null=True, default=None)
+    datafile = serializers.JSONField(source="serialized_data_file")
+    key = serializers.JSONField(source="data_file_key")
+
+    class Meta:  # noqa: D101
+        model = NewDataFileAccessLog
+        fields = ["date", "ip_address", "user", "datafile", "key", "aws_url"]
+
+
+class AWSDataFileAccessLogSerializer(serializers.ModelSerializer):
+    """
+    Serialize the logs of file access events from Amazon's end.
```

Comment:
`Serialize logs of AWS file access events for custom API endpoint for OHLOG_PROJECT_ID`